File: 07. Perulangan Lanjut/A. Break Continue Exit/a_break_continue_exit.c
#include <stdio.h>
int main(){
  int N, i;
  scanf("%d", &N);
  for (i = 1; i <= N; i++){
    if (i % 10 == 0){
      continue;
    }
    else if (i == 42){
      printf("ERROR\n");
      break;
    }
    else{
      printf("%d\n", i);
    }
  }
  return 0;
}

File: 07. Perulangan Lanjut/A. Break Continue Exit/a_break_continue_exit.rs
fn main() {
    let mut reader = String::new();
    std::io::stdin().read_line(&mut reader).expect("");
    let n: i32 = reader.trim().parse().expect("");
    for i in 1..n + 1 {
        if i % 10 == 0 {
            continue;
        } else if i == 42 {
            println!("ERROR");
            break;
        } else {
            println!("{}", i);
        }
    }
}
